inst/sensorreading/sensorreading.Rmd

```diff
@@ -62,8 +62,11 @@ if(!identical(tbl, "The dataset requested is empty.")){
 ***
 
 ```{r,echo=FALSE,result='asis',comment=NA,warning=FALSE}
+library(dplyr)
 if(!identical(tbl, "The dataset requested is empty.")){
-  quals <- as.data.frame(data$readings$qualifiers)
+  quals <- bind_rows(lapply(data$readings$qualifiers, function(qualRow) {
+                    as.data.frame(Filter(Negate(is.null), qualRow))
```

Comment:
instead of `Negate`, we use `!is.null()` (I believe that's what Negate is doing)